doc/source/contribute.rst

```diff
@@ -132,6 +261,84 @@ For example: ::
       return str(number) == string
 
 
+Step 4: Test your code
+~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes doctests are not enough and new features require to go a step further by writing unit tests.
+Our unit tests modules are located in `/larray/tests/`.
+See the :ref:`Tests <contributing.testing>` section bellow for more details.
+
+Step 5: Add a change log
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Changes should be reflected in the release notes located in ``doc/source/changes/version_<next_release_version>.inc``.
+This file contains an ongoing change log for the next release.
+Add an entry to this file to document your fix, enhancement or (unavoidable) breaking change.
+If you hesitate in which section to add your change log, feel free to ask.
+Make sure to include the GitHub issue number when adding your entry (using `` closes :issue:`issue-number` ``
+where `issue-number` is the number associated with the fixed issue).
+
+Step 6: Commit your code
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you think you have (finally) fixed the issue (after documenting your code, running all the tests
```

Comment:
s/When you think you have (finally) fixed the issue (after documenting your code, running all the tests and adding a change log), m/When all the above is done, commit your changes. M/